src/features/presence/PresenceAvatarList.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import noop from 'lodash/noop';
+// @ts-ignore flow import
+import PresenceAvatar from './PresenceAvatar';
+import PresenceAvatarTooltip from './PresenceAvatarTooltip';
+import Tooltip, { TooltipPosition } from '../../components/tooltip';
+import './PresenceAvatarList.scss';
+
+export type Collaborator = {
+    avatarUrl?: string;
+    id: string;
+    isActive: boolean;
+    interactedAt: number;
+    interactionType: string;
+    name: string;
+    profileUrl?: string;
+};
+
+export type Props = {
+    avatarAttributes?: React.HTMLAttributes<HTMLDivElement>;
+    className?: string;
+    collaborators: Array<Collaborator>;
+    hideTooltips?: boolean;
+    maxAdditionalCollaborators?: number;
+    maxDisplayedAvatars?: number;
+    onAvatarMouseEnter?: (id: string) => void;
+    onAvatarMouseLeave?: () => void;
+};
+
+function PresenceAvatarList(props: Props, ref: React.Ref<HTMLDivElement>): JSX.Element | null {
+    const {
+        avatarAttributes,
+        className,
+        collaborators,
+        hideTooltips,
+        maxAdditionalCollaborators = 99,
+        maxDisplayedAvatars = 3,
+        onAvatarMouseEnter = noop,
+        onAvatarMouseLeave = noop,
+        ...rest
+    } = props;
+
+    const [activeTooltip, setActiveTooltip] = React.useState<string | null>(null);
+
+    const hideTooltip = (): void => {
+        setActiveTooltip(null);
+        if (onAvatarMouseLeave) {
+            onAvatarMouseLeave();
+        }
+    };
+
+    const showTooltip = (id: string): void => {
+        setActiveTooltip(id);
+        if (onAvatarMouseEnter) {
+            onAvatarMouseEnter(id);
+        }
+    };
+
+    if (!collaborators.length) {
+        return null;
+    }
+
+    return (
+        <div ref={ref} className={classNames('bdl-PresenceAvatarList', className)} {...rest}>
+            {collaborators.slice(0, maxDisplayedAvatars).map(collaborator => {
+                const { id, avatarUrl, name, isActive, interactedAt, interactionType } = collaborator;
+                return (
+                    <Tooltip
+                        key={id}
+                        isShown={!hideTooltips && activeTooltip === id}
+                        position={TooltipPosition.BOTTOM_CENTER}
+                        text={
+                            <PresenceAvatarTooltip
```

Comment:
The naming here is a little confusing, since this isn't actually a Tooltip component. Should the Tooltip wrapper move into PresenceAvatarTooltip?